README.md

```diff
@@ -98,163 +98,6 @@ mix test --only integration
-
-```elixir
-tx_bytes = 
-  <<248, 77, 1, 192, 245, 244, 1, 148, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0, 148, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-    0, 0, 136, 0, 0, 0, 0, 0, 0, 0, 100, 148, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0, 0, 0>>
-
-ExPlasma.Transaction.decode(tx_bytes)
-
-#=>
-%ExPlasma.Transaction{
-  inputs: [],
-  metadata: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0>>,
-  outputs: [
-    %ExPlasma.Utxo{
-      amount: <<0, 0, 0, 0, 0, 0, 0, 100>>,
-      blknum: 0,
-      currency: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0>>,
-      oindex: 0,
-      output_type: 1,
-      owner: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0>>,
-      txindex: 0
-    }
-  ],
-  sigs: []
-}
-```
-
-### Generating and Submitting Blocks
-#### Creating a block
-
-```elixir
-utxo = %Utxo{blknum: 1}
-output = %Utxo{owner: <<0::160>>, amount: 1}
-Payment.new(%{inputs: [utxo], outputs: [output]})
-|> List.wrap()
-|> Block.new()
-#=> 
-%ExPlasma.Block{
-  hash: <<1, 16, 81, 170, 227, 107, 163, 227, 126, 169, 45, 179, 200, 94, 182,
-    192, 241, 138, 24, 219, 149, 122, 252, 243, 115, 35, 14, 69, 152, 11, 98,
-    200>>,
-  timestamp: nil,
-  transactions: [
-    %ExPlasma.Transactions.Payment{
-      inputs: [
-        %ExPlasma.Utxo{
-          amount: 0,
-          blknum: 1,
-          currency: "0x0000000000000000000000000000000000000000",
-          oindex: 0,
-          output_type: 1,
-          owner: "0x0000000000000000000000000000000000000000",
-          txindex: 0
-        }
-      ],
-      metadata: nil,
-      outputs: [
-        %ExPlasma.Utxo{
-          amount: 1,
-          blknum: 0,
-          currency: "0x0000000000000000000000000000000000000000",
-          oindex: 0,
-          output_type: 1,
-          owner: <<0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0>>,
-          txindex: 0
-        }
-      ],
-      sigs: []
-    }
-  ]
-}
-```
-
-#### Submitting a block to the contract
-
-```elixir
-utxo = %Utxo{blknum: 1}
-output = %Utxo{owner: <<0::160>>, amount: 1}
-{:ok, _receipt_hash} =
-Payment.new(%{inputs: [utxo], outputs: [output]})
-|> List.wrap() 
-|> Block.new()
-|> Client.submit_block()
-```
-
-### Standard Exits
-#### Starting a standard exit
-
-These are more complicated flows that require you to have a valid transaction
-already existing on the network. You can see the [integration tests](test/integration/standard_exit_test.exs) for an example how to achieve the full
-flow. If you were to however have the data on hand, you can pass it to the contract as such:
-
-```elixir
-Client.start_standard_exit(tx_bytes, %{
- from: authority_address(),
- utxo_pos: utxo_pos,
- proof: proof
-})
-```
-#### Processing a standard exit
-#### Challenge a standard exit
-
-### InFlight Exits
-#### Starting an in-flight exit
-#### Processing an in-flight exit
-#### Challenge an in-flight exit
-
```

Comment:
Removing this section of the `README` as this information is available(and more up to date) with the inline hex docs.